Altis_Life.Altis/description.ext

```diff
@@ -10,7 +10,7 @@ unsafeCVL = 1; // Allows createVehicleLocal to be executed in Multiplayer
 class CfgServer {
     DatabaseName = "altislife"; //Config name that'll be grep in the extdb-conf.ini. Default: [altislife]
     DebugMode = 0; //Enable many server/hc debugging logs. Default: 0 (1 = Enabled / 0 = Disabled)
-    HeadlessSupport = 0; //Enable/Disable Headless client support. Default: 1 (1 = Enabled / 0 = Disabled)
+    HeadlessSupport = 1; //Enable/Disable Headless client support. Default: 1 (1 = Enabled / 0 = Disabled)
```

Comment:
Should be disabled by default

Reply:
in fact using HC should be more encouraged as it significantly reduce system stress
on my root (2x some 4-core xeons) it reduced from single-thread 85% to server about 35% and the HC to about 20% over at least two cores (which a dual-cpu system can really benefit from when both process split not just between cores of the same physical chip but between both chips) - hence I left this in intentional

Reply:
Whilst what you're saying is true. Most server owners do not enable HC due to the extended setup it takes. Which is why it has been disabled by default